shared/chat/conversation/info-panel/index.js

```diff
@@ -212,68 +213,134 @@ type BigTeamInfoPanelProps = infoPanelProps & {
-        <Button
-          type="Primary"
-          label="Join channel"
-          style={{marginRight: globalMargins.xtiny}}
-          small={true}
-          onClick={props.onJoinChannel}
-        />
-      )}
-      {!props.isPreview && (
-        <Button type="Danger" small={true} label="Leave channel" onClick={props.onLeaveConversation} />
-      )}
-    </Box>
+          <ClickableBox
+            style={{...globalStyles.flexBoxRow, alignSelf: 'center', alignItems: 'center'}}
+            onClick={props.onViewTeam}
+          >
+            <Avatar teamname={props.teamname} size={12} />
+            <Text type="BodySmallSemibold" style={{marginLeft: globalMargins.xtiny}}>
+              {props.teamname}
+            </Text>
+          </ClickableBox>
+
+          {!props.isPreview && (
+            <Box>
+              <Divider style={styleDivider} />
+              <MuteRow muted={props.muted} onMute={props.onMuteConversation} label="Mute entire channel" />
+              <Notifications />
+            </Box>
+          )}
+
+          <Divider style={styleDivider} />
+
+          <Box style={{...globalStyles.flexBoxRow, justifyContent: 'center'}}>
+            {props.isPreview && (
+              <Button
+                type="Primary"
+                label="Join channel"
+                style={{marginRight: globalMargins.xtiny}}
+                small={true}
+                onClick={props.onJoinChannel}
+              />
+            )}
+            {!props.isPreview && (
+              <Button type="Danger" small={true} label="Leave channel" onClick={props.onLeaveConversation} />
+            )}
+          </Box>
+
+          {props.isPreview && (
+            <Text type="BodySmall" style={{textAlign: 'center', marginTop: globalMargins.xtiny}}>
+              Anyone in {props.teamname} can join.
+            </Text>
+          )}
+
+          <Divider style={styleDivider} />
+          <Box style={{...globalStyles.flexBoxRow, marginRight: globalMargins.small}}>
+            <Text style={{flex: 1, paddingLeft: globalMargins.small}} type="BodySmallSemibold">
+              In this channel ({props.participants.length.toString()})
+            </Text>
+            {props.admin &&
+              props.channelname === 'general' && (
+                <Text type="BodySmallPrimaryLink" onClick={props.onViewTeam}>
+                  Manage
+                </Text>
+              )}
+          </Box>
+        </Box>
+      )
+    case 'participant':
+      return (
+        <Participant key={props.key} participant={props.participant} onShowProfile={props.onShowProfile} />
+      )
+  }
+}
 
-    {props.isPreview && (
-      <Text type="BodySmall" style={{textAlign: 'center', marginTop: globalMargins.xtiny}}>
-        Anyone in {props.teamname} can join.
-      </Text>
-    )}
+const _rowSizeEstimator = (index, cache) => {
+  if (index === 0) {
+    return cache[1] || 429 // estimate basd on size of header in non-admin non-preview mode
```

Comment:
basd -> based
also why 1, if its something weird in react list we should put a comment

Reply:
So turns out cache is `{[index: number]: (height: number)}` (which for some reason is an object instead of an array). Changed this to access `cache[0]`

Reply:
thanks for pointing it out!